README.md

```diff
@@ -1 +1,84 @@
-# ride-share-two
\ No newline at end of file
+# Ride Share
+Remember the ride share exercise we did with designing and creating a system to track the ride share data from a CSV file? We did a lot of great work on this exercise in creating arrays and hashes of data, but we've learned a lot since we did that exercise! Now, we're going to use our understanding of classes, methods and attributes to create an object-oriented implementation of our ride share system.
+
+## Learning Goals
+Reinforce and practice all of the Ruby and programming concepts we've covered in class so far:
+- Creating and instantiating classes with attributes
+- Creating class and instance methods within our classes
+- Writing pseudocode and creating tests to drive the creation of our code
+- Importing data from a CSV file
+
+## Primary Requirements
```

Comment:
I think the requirements section could be structured a bit more, by having multiple levels in the requirement lists.

Where we're saying "You should be able to" multiple times in a row we can use that prefix once and nest the various requirements under it.